### ISSUE TITLE
Sample wallet address hardcoded in tutorial script - Issuing Assets

### ISSUE THREAD

**Issue by adiack** [ElementsProject/elementsproject.github.io#39]
In script runtutorial.sh a sample address is hardcoded which will result in line not working.

```
###### ASSETS #######

e1-cli getwalletinfo

e1-cli getwalletinfo bitcoin

e1-cli dumpassetlabels

e1-cli getwalletinfo b2e15d0d7a0c94e4e2ce0fe6e8691b9e451377f6e46e8045a86f7c4b5d4f0f23
```
Could be changed to

```
###### ASSETS #######

e1-cli getwalletinfo

e1-cli getwalletinfo bitcoin

e1-cli dumpassetlabels

e1-cli getwalletinfo $(e1-cli dumpassetlabels | jq '.bitcoin' | tr -d '"')
```

**Comment by wintercooled** [comment]
I think this asset ID will always be the same because of the way it is generated on chain initialisation - but I like your approach so I will make a PR for the change after testing locally. Thanks @adiack

**Comment by wintercooled** [comment]
I merged PR #40 to be sure.

**Comment by wintercooled** [comment]
Also PR #41 relates to this - added to the main tutorial code that the easy run code .sh is based upon with a bit of explanatory detail.
